plugins/frontend/backstage-plugin-github-insights/src/components/Widgets/LanguagesCard/LanguagesCard.tsx

```diff
@@ -72,22 +72,19 @@ const LanguagesCard = () => {
   const { owner, repo } = useProjectEntity(entity);
   const { value, loading, error } = useRequest(entity, 'languages', 0, 0);
   const projectAlert = isGithubInsightsAvailable(entity);
+  const alertApi = useApi(alertApiRef);
+
   if (!projectAlert) {
     return (
       <MissingAnnotationEmptyState annotation={GITHUB_INSIGHTS_ANNOTATION} />
     );
   }
 
-  if (loading) {
-    return <Progress />;
-  } else if (error) {
-    return (
-      <Alert severity="error" className={classes.infoCard}>
-        {error.message}
-      </Alert>
-    );
+  if (loading) return <Progress />;
+  if (error) {
+    alertApi.post({ message: error.message, severity: 'error' });
```

Comment:
Will it be obvious which of the many plugins rendered to the page, which plugin is erroring here?